### PR TITLE
BC-2762 - rename ukrainian language key ua to uk

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Short Description

https://ticketsystem.dbildungscloud.de/browse/BC-2762

- https://github.com/hpi-schul-cloud/dof_app_deploy/pull/526
- https://github.com/hpi-schul-cloud/schulcloud-server/pull/4139
- https://github.com/hpi-schul-cloud/schulcloud-client/pull/3199
- https://github.com/hpi-schul-cloud/end-to-end-tests/pull/438

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/date-time-picker/DatePicker.vue [hunk 1]
```diff
@@ -76,12 +76,11 @@ export default defineComponent({
...
-			if (i18n.locale === "ua") {
-				return "uk";
-			}
-
-			return i18n.locale;
+			// 	// TODO remove if language code for ukraine is fixed
+			// 	if (i18n.locale === "ua") {
+			// 		return "uk";
+			// 	}
+			// 	return i18n.locale;
```
Comment (MartinSchuhmacher) [inline src/components/date-time-picker/DatePicker.vue:83]: delete whole function if not needed anymore

File: src/components/date-time-picker/DateTimePicker.vue [hunk 2]
```diff
@@ -67,10 +67,10 @@ export default defineComponent({
...
 
 		const locale = (() => {
-			// TODO remove if language code for ukraine is fixed
-			if (i18n.locale === "ua") {
-				return "uk";
-			}
+			// 	// TODO remove if language code for ukraine is fixed
+			// 	if (i18n.locale === "ua") {
+			// 		return "uk";
+			// 	}
```
Comment (MartinSchuhmacher) [inline src/components/date-time-picker/DateTimePicker.vue:73]: remove comment/restructure function call

File: src/components/editor/CKEditor.vue [hunk 3]
```diff
@@ -51,11 +51,11 @@ export default defineComponent({
...
-			// map ua to correct uk
-			// TODO remove if language code is fixed
-			if (i18n.locale === "ua") {
-				return "uk";
-			}
+			// 	// map ua to correct uk
+			// 	// TODO remove if language code is fixed
+			// 	if (i18n.locale === "ua") {
+			// 		return "uk";
+			// 	}
```
Comment (MartinSchuhmacher) [inline src/components/editor/CKEditor.vue:58]: remove comment/restructure function call

File: src/pages/files/FilesOverview.page.vue [hunk 4]
```diff
@@ -85,9 +85,9 @@ export default defineComponent({
 		const breadcrumbs: Ref<Breadcrumb[]> = ref(filesPage.breadcrumbs);
 
 		const locale = (): Locale => {
-			if (i18n.locale === ChangeLanguageParamsLanguageEnum.Ua) {
-				return "uk"; // TODO https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes use everywhere uk (language code)
-			}
+			// if (i18n.locale === ChangeLanguageParamsLanguageEnum.Uk) {
+			// 	return "uk"; // TODO https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes use everywhere uk (language code)
+			// }
```
Comment (MartinSchuhmacher) [inline src/pages/files/FilesOverview.page.vue:90]: remove comment/restructure function call

**Review by MartinSchuhmacher — APPROVED** [review]

**Review by christian-darsow — APPROVED** [review]

**Pull request merged**